quadratic/quadratic.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func main() {
+	fmt.Println("The program finds solutions of a quadratic equation.")
+	fmt.Println("Enter the coefficients A, B and C:")
+	var a, b, c float64
+	fmt.Scanln(&a, &b, &c)
+	D := b*b - 4*a*c
+	x1 := (-b + math.Sqrt(D)) / 2 * a
```

Comment:
That's a bug here. Check this: `10/2*5` the result is 25 (10/2=5, and 5*5=25). If you want to get 1, you have to put 2*5 into parentheses